vendor/engines/split_accounts/spec/services/order_detail_splitter_spec.rb

```diff
@@ -1,77 +1,134 @@
 require "rails_helper"
 
 RSpec.describe SplitAccounts::OrderDetailSplitter, type: :service do
-  let(:subaccount_1) { build_stubbed(:nufs_account) }
-  let(:subaccount_2) { build_stubbed(:nufs_account) }
+  describe "item/service orders" do
```

Comment:
I think it's fair to say using a more explicit name isn't going to work.  :smile: